lilypad/generations.py

```diff
@@ -71,23 +94,13 @@ async def inner_async(*args: _P.args, **kwargs: _P.kwargs) -> _R:
                     f"{fn.__name__}"
                 ) as span:
                     output = await fn(*args, **kwargs)
-                    results = str(output)
                     if isinstance(output, BaseModel):
```

Comment:
move this section inside `_contruct_trace_attributes` since its shared for async